src/JSHandle.ts

```diff
@@ -291,7 +291,24 @@ export class ElementHandle extends JSHandle {
     // the cost unnecessarily.
     // eslint-disable-next-line @typescript-eslint/no-var-requires
     const path = require('path');
-    const files = filePaths.map(filePath => path.resolve(filePath));
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const fs = require('fs');
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const {promisify} = require('util');
+    const access = promisify(fs.access);
```

Comment:
Should we use `fs.constants.R_OK` to ensure the file doesn't just exist but also that it is readable? WDYT?

Reply:
Done